docs/ref/forms/api.txt

```diff
@@ -932,6 +932,50 @@ and using the template above, would render something like:
 
     <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" />
 
+.. _customizing-bound-field:
+
+Customizing ``BoundField``
+--------------------------
+
+.. versionadded:: 1.9
+
```

Comment:
`.. method:: Form.bind_to_form()`
(I'd link that in the release notes instead of the section heading.)